### PR TITLE
fr_FR : All po files

### PR DESCRIPTION
**Pull request by tematim**
Hi,

This PR is an update will all po files in fr-FR language.

Regards

### CODE REVIEW

File: Localization/fr-FR/about.po [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: fr_FR\n"
+"X-Generator: Poedit 2.3.1\n"
+
+msgid "À propos"
+msgstr ""
```
Comment (Tar-Minyatur) [inline Localization/fr-FR/about.po]: The idea is to keep the `msgid` the original and put the translation into the `msgstr`:
```suggestion
msgid "About"
msgstr "À propos"
```

**Review by Tar-Minyatur — CHANGES_REQUESTED** [review]
The header data looks perfectly fine. But there is an issue with the content. See the comment I added directly in the file.

File: Localization/fr-FR/about.po (hunk 1 above)
Comment (Tar-Minyatur) [reply Localization/fr-FR/about.po]: Just to point it out, this was just one example. :) Of course the same has to be done for all the other entries.

Comment (tematim) [reply Localization/fr-FR/about.po]: Is it possible to review again ?

**Review by Tar-Minyatur — APPROVED** [review]
Looks good to me. 👍

**Review by fangfufu — APPROVED** [review]

**Comment by tematim** [comment]
To link with the ticket #8

**Pull request merged**